elide-integration-tests/src/test/java/com/yahoo/elide/async/integration/tests/TableExportIT.java

```diff
@@ -0,0 +1,1250 @@
+                                                field("query"),
+                                                field("queryType"),
+                                                field("status")
+                                        )
+                                )
+                        )
+                )
+        ).toQuery();
+
+        JsonNode graphQLJsonNode = toJsonNode(graphQLRequest, null);
+        ValidatableResponse response = given()
+                .contentType(MediaType.APPLICATION_JSON)
+                .accept(MediaType.APPLICATION_JSON)
+                .body(graphQLJsonNode)
+                .post("/graphQL")
+                .then()
+                .statusCode(org.apache.http.HttpStatus.SC_OK);
+
+        String output = response.extract().body().asString();
+        assertEquals(true, output.contains("errors"));
+        assertEquals(true, output.contains("CreatePermission Denied"));
+    }
+
+    /**
+     * Test for ResultType Set to Invalid.
+     */
+    @Test
+    public void graphQLTestCreateFailOnResultType() {
+
+        TableExport queryObj = new TableExport();
+        queryObj.setId("edc4a871-dff2-4054-804e-d80075cf939e");
+        queryObj.setAsyncAfterSeconds(0);
+        queryObj.setQueryType("GRAPHQL_V1_0");
+        queryObj.setStatus("QUEUED");
+        queryObj.setResultType("XLS");
+        queryObj.setQuery("{\"query\":\"{ book { edges { node { id title } } } }\",\"variables\":null}");
+        String graphQLRequest = document(
+                mutation(
+                        selection(
+                                field(
+                                        "tableExport",
+                                        arguments(
+                                                argument("op", "UPSERT"),
+                                                argument("data", queryObj, UNQUOTED_VALUE)
+                                        ),
+                                        selections(
+                                                field("id"),
+                                                field("query"),
+                                                field("queryType"),
+                                                field("status")
+                                        )
+                                )
+                        )
+                )
+        ).toQuery();
+
+        JsonNode graphQLJsonNode = toJsonNode(graphQLRequest, null);
+        ValidatableResponse response = given()
+                .contentType(MediaType.APPLICATION_JSON)
+                .accept(MediaType.APPLICATION_JSON)
+                .body(graphQLJsonNode)
+                .post("/graphQL")
+                .then()
+                .statusCode(org.apache.http.HttpStatus.SC_OK);
+
+        String output = response.extract().body().asString();
+        assertEquals(true, output.contains("errors"));
+        assertEquals(true, output.contains("data.resultType"));
```

Comment:
Is this the error?  THis is pretty cryptic.

Reply:
It's a partial error message. I am just making sure the error message contains resultType to confirm a value of resultType caused this error.